README.md

```diff
@@ -259,17 +258,39 @@ will be converted to `width="100"`.
 The class can be used like this:
 
 ```php
-$converter = new \Pelago\Emogrifier\HtmlProcessor\CssToAttributeConverter($rawHtml);
+$converter = \Pelago\Emogrifier\HtmlProcessor\CssToAttributeConverter::fromHtml($rawHtml);
+$visualHtml = $converter->convertCssToVisualAttributes()->render();
+```
+
+You can also have the `CssToAttributeConverter` work on a `DOMDocument`:
+
+```php
+$converter = \Pelago\Emogrifier\HtmlProcessor\CssToAttributeConverter::fromDomDocument($domDocument);
 $visualHtml = $converter->convertCssToVisualAttributes()->render();
 ```
 
 ### Technology preview of new classes
 
 Currently, a refactoring effort is underway, aiming towards replacing the
 grown-over-time `Emogrifier` class with the new `CssInliner` class and moving
-additional HTML processing into separate `CssProcessor` classes (which will
-inherit from `AbstractHtmlProcessor`). You can try the new classes, but be
-aware that the APIs of the new classes still are subject to change. 
+additional HTML processing into separate `CssProcessor` classes (which
+inherits from `AbstractHtmlProcessor`). You can try the new classes, but be
```

Comment:
should be "inherit" (3rd person plural - a class inherits, but classes inherit).

Reply:
Done.